src/molecules/header-card/header-card.tsx

```diff
@@ -37,8 +37,20 @@ const HeaderCardContainer = styled.div`
 export const DefaultHeaderCard = ({ story, config, storyType }: HeaderCardProps) => {
   const { publisherConfig } = config;
 
+  const theme = useTheme();
+
   const { enableLastPublished, enableFirstPublished } = getDateSettings(config, storyType);
 
+  function authorCardRender(data) {
+    let authorUi;
+    const authorCard = get(config, ["opts", "featureConfig", "renderAuthorCard"], null);
```

Comment:
pls rename `renderAuthorCard` to `authorCardRender` as that is a convention amplib follows

Reply:
done